_books/ion-1-1/src/glossary.md

```diff
@@ -0,0 +1,160 @@
+# Glossary
+
+**active encoding module**<br/>
+The _encoding module_ whose symbol table and macro table are available in the current _segment_ of an Ion _document_.
+The active encoding module is set by a _directive_.
```

Comment:
Thinking out loud, it would be really nice to incorporate something like [this pre-processor](https://github.com/caukub/mdbook-hints) to make phrases in the spec show their definition on hover.

Having a glossary is great, but not having to leave the page to define a term would be even better.

EDIT: I just saw that preprocessor's [first and only issue](https://github.com/caukub/mdbook-hints/issues/1). 😆